<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkBusinessActionWidget</class>
 <widget class="QWidget" name="BookmarkBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QWidget" name="recordAfterWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="recordAfterSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordAfterSuffixLabel">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="durationWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="durationSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationSuffixLabel">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="recordBeforeWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="recordBeforeSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="recordBeforeSuffixLabel">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="tagsLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="recordBeforeLabel">
     <property name="text">
      <string>Pre-recording:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="recordAfterLabel">
     <property name="text">
      <string>Post-recording:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="fixedDurationCheckBox">
     <property name="text">
      <string>Fixed duration:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>Tags:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
